operators/alibi/index.ts

```diff
@@ -3,7 +3,7 @@ import { IOperator } from "~/types/operator"
 export const alibi: IOperator = {
   name: "Alibi",
   role: "Defender",
-  unit: "GIS",
+  unit: "VIPERSTRIKE",
```

Comment:
This is wrong, `unit` value should remain unmodified, instead we need to add new `squad` value